frontend/taipy-gui/src/context/taipyReducers.ts

```diff
@@ -284,24 +286,46 @@ const initializeBroadcastManagement = (dispatch: Dispatch<TaipyBaseAction>) => {
     }, broadcast_timeout);
 };
 
+// App id
+const checkAppId = (appId: string) => {
+    if (!appId) {
+        return;
+    }
+    appId = `${appId}`;
```

Comment:
`appId= appId.toString();` ?

Reply:
`${...}` is broader

Reply:
It sure is. But why should it be? Certainly far slower don't you think?

Reply:
I was more interested in your advice on the naming of the config option (app_id)